scRNA-seq/02-tag-based_pre-processing_scRNA-seq.md

```diff
@@ -19,12 +21,10 @@ Salmon, just like was done in the bulk RNA-seq module.
 
 ### Step 0: Obtaining the data
 
-10X Genomics has plenty of [datasets](https://support.10xgenomics.com/single-cell-gene-expression/datasets)
-available to play around with.
-For this example, we will use the [1k pbmc](https://support.10xgenomics.com/single-cell-gene-expression/datasets/3.0.0/pbmc_1k_v2).
-We will process a fastq file of this as an example.  
+We obtained these data from Tabula Muris project's [Figshare](https://figshare.com/projects/Tabula_Muris_Transcriptomic_characterization_of_20_organs_and_tissues_from_Mus_musculus_at_single_cell_resolution/27733) 
```

Comment:
Did you end up getting this from Figshare or did you get it from AWS and then convert from BAM files? I may be mixing this up with another dataset. I'm essentially after whether or not someone could go to the figshare and easily obtain the exact file you are providing.

Reply:
I ended up getting bam files from Figshare and then converting them to fastq files using `CellRanger bamtofastq`

Reply:
I will add this note so no one is potentially confused.